config/env_blank.json

```diff
@@ -13,9 +13,14 @@
   "CANVAS_URL": "",
   "CANVAS_TOKEN": "",
   "NUM_ASYNC_WORKERS": 8,
-  "ZOOM_TOKEN": "<ADD THIS TOKEN>",
-  "ZOOM_BASE_URL": "<ADD zoom.us URL>",
-  "ZOOM_EARLIEST_FROM": "YYYY-MM-DD",
+  "ZOOM_CONFIG": {
```

Comment:
Yeah, I think this is the right place in the ordering for the `README`.

Reply:
Yeah I wanted to do a table within a table (or something) in the README. We didn't have anything similar. If it was YAML we could have just commented right here! ;)

Reply:
As I think I've said before, I'm fine with switching to YAML, but having a separate resource still seems valuable to me.